src/Command/RunCommand.php

```diff
@@ -213,7 +213,7 @@ protected function configure(): void
         ;
     }
 
-    protected function executeCommand(IO $io): void
+    protected function executeCommand(IO $io): bool
```

Comment:
how about keeping `void` here? In practice we tend to throw an exception in case of an error instead of a boolean value

Reply:
Do you propose that we re-throw the exceptions?

Reply:
We actually already do since we don't wrap the whole call in with a `catch (Throwable $t)` (and Symfony already does it). So I don't think forcing this return code is providing anything tbh

Reply:
Well, we do wrap the other call for two very important exceptions.